SPECS/kernel-uvm/config

```diff
@@ -759,8 +735,6 @@ CONFIG_ARCH_HAS_GCOV_PROFILE_ALL=y
 # end of GCOV-based kernel profiling
 
 CONFIG_HAVE_GCC_PLUGINS=y
```

Comment:
Should CONFIG_GCC_PLUGINS and/or CONFIG_ARCH_HAS_CC_PLATFORM be kept, using the newer AzL compiler?